src/UI/Root/Components/Header/Header.tsx

```diff
@@ -34,4 +34,24 @@ const StyledImage = styled(SimpleBackgroundImage)`
 
 const StyledHeader = styled(PageHeader)`
   background-color: transparent;
+  //var(--pf-global--breakpoint--md) doesn't seems work
```

Comment:
but isn't this a value you need to assign to a css prop?

Reply:
but that is patternfly variable, it should be available out of the box as other variables used in the project